eva/models/storage/batch.py

```diff
@@ -270,3 +276,27 @@ def reverse(self):
     def reset_index(self):
         """ Resets the index of the data frame in the batch"""
         self._frames.reset_index(drop=True, inplace=True)
+
+    def modify_column_alias(self, alias: str) -> NoReturn:
+        # a, b, c -> table1.a, table1.b, table1.c
+        # t1.a -> t2.a
+        new_col_names = []
+        for col_name in self.frames.columns:
+            if '.' in col_name:
+                new_col_names.append('{}.{}'.format(
+                    alias, col_name.split('.')[1]))
```

Comment:
We will change this to be index-based later right? This feels like a hack method.

Reply:
Yeah, I know. There was a lot to change so went with this design. Surely, we got to change it to index. I haven't thought of how we would do it.